### PR TITLE
Adds self-verify functionality

### PR DESCRIPTION
**Pull request by nrempel**

### CODE REVIEW

**Comment by nrempel** [comment]
This depends on this feature: https://github.com/hyperledger/aries-cloudagent-python/pull/181

Once that is merge, we will need to cut a new release of aca-py and update this repo to build from the new release

**Comment by esune** [comment]
> 
> 
> This depends on this feature: [hyperledger/aries-cloudagent-python#181](https://github.com/hyperledger/aries-cloudagent-python/pull/181)
> 
> Once that is merge, we will need to cut a new release of aca-py and update this repo to build from the new release

@andrewwhitehead please let me know when aca-py 0.3.3 is out, and I will release a new version of `aries-cloudagent-container` so that we can re-build and test these changes.

File: starter-kits/credential-registry/server/tob-api/api_v2/views/rest.py [hunk 1]
```diff
@@ -210,6 +221,81 @@ def retrieve_formatted(self, request, pk=None):
     #    serializer = CredentialSerializer(queryset, many=True)
     #    return Response(serializer.data)
 
+    @detail_route(url_path="verify", methods=["get"])
+    def verify(self, request, pk=None):
+        item = self.get_object()
+
+        connection_response = requests.get(
+            f"{AGENT_ADMIN_URL}/connections?alias={django.conf.settings.AGENT_SELF_CONNECTION_ALIAS}"
```
[the remark this answers is not in the document]
Comment (nrempel) [reply starter-kits/credential-registry/server/tob-api/api_v2/views/rest.py]: good catch

File: starter-kits/credential-registry/server/tob-api/api_v2/views/rest.py [hunk 2]
```diff
@@ -210,6 +221,81 @@ def retrieve_formatted(self, request, pk=None):
...
+        response = requests.get(
+            f"{AGENT_ADMIN_URL}/credential_exchange/{item.credential_exchange_id}"
+        )
+        response_body = response.json()
+
+        assert response_body["state"] == "stored"
+
+        presentation_request = {
+            "version": "1.0",
+            "name": "self-verify",
```
[the remark this answers is not in the document]
Comment (nrempel) [reply starter-kits/credential-registry/server/tob-api/api_v2/views/rest.py]: At the moment, they won't change. When handling the verification, the webhook handler is designed to handle any proof request.

**Comment by WadeBarnes** [comment]
@esune, Please confirm whether or not your requests have been addressed.

**Comment by nrempel** [comment]
Caught me while I was on lunch! The latest commit updates the image.

**Comment by nrempel** [comment]
I've tested this locally and everything works.

We are still "faking" the verified response because of the bug in libindy. Once this change has been merged and released, we can update to remove the workaround:
https://github.com/hyperledger/indy-sdk/pull/1893

**Comment by esune** [comment]
Looks good to me. I see we also already have an issue logged to track this ( #349 ). 👍

**Comment by esune** [comment]
@WadeBarnes this is ready to go once your request for changes is dismissed.

**Pull request merged**